components/ntp_background_images/browser/ntp_background_images_service.cc

```diff
@@ -189,8 +189,8 @@ void NTPBackgroundImagesService::RegisterSponsoredImagesComponent() {
 
   const auto data = GetSponsoredImagesComponentData(country_code);
   if (!data) {
-    DVLOG(2) << __func__ << ": Not support NTP SI component for "
-             << country_code;
+    VLOG(6) << "NTP Sponsored Images component is not supported for "
```

Comment:
`unsupported`

Reply:
VLOG(6) << "NTP Sponsored Images component is not supported in "